README.md

```diff
@@ -2,31 +2,170 @@
+</project>
+```
+
+Don't forget to reload Maven after making changes to `pom.xml`:
 
-## Topic 1
+![reload maven icon](https://curriculum-content.s3.amazonaws.com/6036/create-webapp-project/reload_maven.png)
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
 
-## Topic 2
+## Create a new servlet class named `GreetServlet`
+
+Now we can create a new Java class named `GreetServlet`.  Right-click on the `java` directory
+and create the new class named `GreetServlet`.  Your project should look like this:
+
+![New Servlet Class](https://curriculum-content.s3.amazonaws.com/6036/java-mod-5-servlet-class/greetservlet_class.png)
+
+Copy the following code into `GreetServlet`:
+
+```java
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet(urlPatterns = "/greet")
+public class GreetServlet extends HttpServlet {
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        // Prepare the server response
+        resp.setContentType("text/html; charset=UTF-8");
+        PrintWriter out = resp.getWriter();
+        out.println("<html>");
+        out.println("<head><title>Servlet Example</title></head>");
+        out.println("<body>");
+        out.println("<h1>Greetings!</h1>");
+        out.println("</body>");
+        out.println("</html>");
+    }
+
+}
+```
+
+Let's explore what the code does:
+
+1. The code `@WebServlet(urlPatterns = "/greet")`  tells the servlet container that the servlet
+   will handle requests for the path **http://localhost:8080/greet**.
+2. The code `public class GreetServlet extends HttpServlet ` makes our class a servlet that
+   can override inherited methods from `HttpServlet` to handle HTTP requests.
+3. We override the `doGet()` method to handle incoming **HTTP GET** requests to the URL **http://localhost:8080/greet**.
+   - The method takes two parameters:
+     - `HttpServletRequest req` stores information about the incoming HTTP GET request from the client.
+     - `HttpServletResponse resp` will be used to generate an HTTP response that will be sent to the client.
+        We can see the `doGet()` method create a simple HTML web page as a response.
+
+## Restart Tomcat
+
+While the embedded Tomcat server recognizes and reloads changes
+to the JSP files in the webapp folder, we will need to stop the server
+and restart it for changes made to the servlet class.
+
+1. Stop executing the web application by pressing the red stop icon on the Maven toolbar:   
+   ![stop Tomcat server](https://curriculum-content.s3.amazonaws.com/6036/java-mod-5-servlet-class/stop_server.png)  
```

Comment:
Looks like there might be a weird spacing / formatting thing going on here; at least on GitHub

Reply:
I think it is fixed

Reply:
Looks good now on GitHub!